tests/subsys/dfu/dfu_target/suit/regenerate_manifests.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
why is this in bash and not Python ?

This will make it hard to run those tests on windows.

Reply:
Existing SUIT tests use the .sh files: https://github.com/nrfconnect/sdk-nrf/tree/main/tests/subsys/suit/manifest_common so it is coherent with all other suit-related suites.

Reply:
This bash file uses another bash file that is already used in suit tests. So if I change it to Python it doesn't change anything regarding Windows tests. @tomchy @ahasztag could you please comment on that?

Reply:
This script is not used by the test itself, thus it does not affect running tests on Windows.
This script is used only by NCS developers to easily and quickly regenerate static content if major changes in SUIT structures are introduced (i.e. change in signing key ID, new version of the SUIT spec).
It was written in bash, because that was the most convenient way for a developer to store somewhere the knowledge how to regenerate those files. It is not intended to be used or executed by the end customer.

Reply:
@tomchy thanks for the details.

> It is not intended to be used or executed by the end customer.

Thanks, but customers has everything available so we should keep this in mind.
But with the extra details, then I don't consider this a blocking comment.